ci/tests.py

```diff
@@ -110,8 +110,14 @@ def test_generate_report(self):
                 'details': {
                     'personnel_code': 'personnel_code',
                     'facility_name': 'facility_name',
+                    'default_addr_type': 'msisdn',
                     'role': 'role',
                     'state': 'state',
+                    'addresses': {
+                        'msisdn': {
+                            '+2341234567890': {}
+                        }
+                    }
                 }
             },
             status=200,
```

Comment:
@DevChima did you mean this `status=200`?

This is our testing framework, essentially we're mocking responses to HTTP requests our applications make as part of a test. So instead of actually talking to the identity store api we capture that request and have the test framework reply with the payload and an HTTP Status of 200. The application then thinks this is the response it gets back from the identity store api and continues. That way we can control what is received as input to the application and predict (and test) the outcomes.